source/appendices/b-changes-from-r5.rst

```diff
@@ -0,0 +1,41 @@
+
+.. The COUNTER Code of Practice Release 5 © 2017-2023 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+|clearpage|
+
+.. _appendix-b:
+
+Appendix B: Changes from Previous Releases
+==========================================
+
+Note: The main Code of Practice document takes precedence in the case of any conflicts between it and this appendix.
+
+Starting with Release 5.0.2 the COUNTER Code of Practice Release 5 is maintained in the GitHub repository `Project-Counter/cop5 <https://github.com/Project-Counter/cop5>`_, and all changes are tracked with GitHub issues and linked pull requests. The `change log <https://github.com/Project-Counter/cop5/blob/5.1/CHANGELOG.rst>`_ lists all changes by type and impact of the change.
+
+
+Primary changes between R5 and R5.1
+"""""""""""""""""""""""""""""""""""
+
+* **Item becoming the unit of reporting** - Improved OA reporting is one of the major objectives of R5.1, which requires reporting at the level of the item. This impacts on reporting of book usage: under R5.1, usage of each Book_Segment (e.g. chapter) MUST be reported as a separate item investigation or request, with multiple item investigations or requests rolled up to 1 title investigation or request per book.
+* **Data_Type** - Many publishers reported difficulties with the restricted list of Data_Types provided in R5, so the list has been expanded and the definitions and use of each Data_Type clarified.
+* **Section_Type** - The impact of the above changes means Section_Type is deprecated in R5.1.
+* **Access_Type** - R5.1 clarifies where and how COUNTER Access_Types apply. R5.1 also introduced revised definitions for Access_Types to better facilitate reporting of open and free materials.
+* **Components** - Required for the Item Report in R5, Components are optional in R5.1 to simplify delivery of Item Reports.
+* **Report headers** - R5.1 added a link to the COUNTER Registry (https://registry.projectcounter.org) to the standard report header in both JSON and tabular reports.
+* **SUSHI URLs** - Starting with Release 5.1, the release version number will need to be included in the SUSHI URL path.
+* **SUSHI authentication** - R5.1 deprecates support for IP-based authentication in the COUNTER_SUSHI API. APIKey may be used if a replacement is desired.
+* **SUSHI endpoints** - The /status endpoint is public in R5.1 to allow users to easily check whether a service is live. The /reports endpoint is extended to return information about the first and last months for which data are available. We are also introducing separate parameters for common extensions.
+* **JSON** - Key changes are upgrading to the OpenAPI3.1/JSON schema; avoiding duplicate item and parent metadata; simplifying the Performance structure; moving multi-value attributes into Performance; and simplifying Type/Value and Name/Value lists.
```

Comment:
There was no agreement whether and how to move attributes into Performance, therefore I haven't looked into that. Maybe we should simply mention that the new format will be more compact and easier to parse (more "JSON-like"), without mentioning the individual changes?